### ISSUE TITLE
Support Open Cap Table format

### ISSUE THREAD

**Issue by stijnbe** [octolane-org/cap.octolane.com#14]
https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF

This format seems to be supported by multiple cap table solutions. This would allow migrating from Carta.

**Comment by md-abdul-halim-rafi** [comment]
Yes, this one matches the Pulley schema also. We can start designing our schema based on it.

**Comment by JSv4** [comment]
Super cool project, folks. One of the OCF devs here. Let me know if we can be helpful. Any interest in joining our bi-weekly engineering team meeting @md-abdul-halim-rafi?

**Comment by coffeewithone** [comment]
We would love to! How do we join? my email is one@octolane.com for your convenience.

**Comment by JSv4** [comment]
Hey @coffeewithone, I'll include you guys on the recurring invite for 2024. Going out today.
